README.md

```diff
@@ -328,6 +328,92 @@ pipeline {
 }
 ```
 
+Here's how you install it using Pipelines:
+
+1. Make sure you have the connection details of your JFrog environment.
+2. Save the JFrog connection details as JFrog Platform Access Token integration named **jfrogPlatform**.
+3. Save your Bitbucket access token as a generic integration named **bitbucket** with the **token** as key to the Bitbucket access token.
+4. Create a Pipelines job in Pipelines with the below pipelines.yml content.
+5. In the pipelines.yml, set the values of all the mandatory variables.
+6. In the pipelines.yml, modify the code inside the `step type`  and `onExecute` according to the Pipelines agent operating system.
```

Comment:
...modify the code inside the `step type`  and `onExecute` according to the Pipelines agent operating system.
-->
...if you're using a Windows agent, modify the code inside the `onExecute` sections as described on the yaml comment.